docs/faq.md

```diff
@@ -1,5 +1,137 @@
+3. Increasing the default CSI sidecar `kube-api-qps` (to 20) and `kube-api-burst` (to 100) for all sidecars ([#1834](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1834))
+	- Each sidecar can now send a burst of up to 100 [queries](https://kubernetes.io/docs/reference/using-api/api-concepts/#api-verbs) to the Kubernetes API Server before throttling itself. It will then allow up to 20 more requests per second until it stops bursting.   
+	- This keeps Kubernetes objects (`PersistentVolume`, `PersistentVolumeClaim`, and `VolumeAttachment`) more synchronized with the actual state of AWS resources, at the cost of increasing the load on the K8s API Server from `ebs-csi-controller` pods when many volume operations are happening at once. 
+4. Increasing the default CSI sidecar `timeout` values (from 15s to 60s) for all sidecars ([#1824](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1824))   
+  - E.g. the [external-attacher](https://github.com/kubernetes-csi/external-attacher?tab=readme-ov-file#csi-error-and-timeout-handling) will now give the driver up to 60s to report an attachment success/failure before retrying a `ControllerPublishVolume` call. Now the external-attacher won't prematurely timeout a `ControllerPublishVolume` call that would've taken 20s before returning a success response.  
+	- This decreases the amount of premature timeouts for CSI RPC calls, which reduces the amount of replay EC2 API requests made by and waited for by the driver (at the cost of a longer delay during a real driver timeout (e.g. network blip leads to lost `ControllerPublishVolume` response))
+## EC2 and K8s CSI Sidecar Throttling Overview
+
+Both the EC2 API and K8s CSI sidecars base their API throttling implementation off of the [token bucket algorithm](https://en.wikipedia.org/wiki/Token_bucket). Both the [Amazon EC2 API Reference](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/throttling.html#throttling-limits-rate-based) and the [K8s documentation definitions of qps and burst](https://kubernetes.io/docs/reference/config-api/apiserver-eventratelimit.v1alpha1/#eventratelimit-admission-k8s-io-v1alpha1-Limit) describe their throttling systems.  
+
+Let's look at an example from [Request throttling for the Amazon EC2 API](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/throttling.html#throttling-limits-rate-based)
+
+<details><summary>Expand for excerpt:</summary>
+
+"With request rate limiting, you are throttled on the number of API requests you make. Each request that you make removes one token from the bucket. For example, the bucket size for _non-mutating_ (`Describe*`) API actions is 100 tokens, so you can make up to 100 `Describe*` requests in one second. If you exceed 100 requests in a second, you are throttled and the remaining requests within that second fail.
```

Comment:
Suggest removing this excerpt from the FAQ section and simply linking to the relevant documentation instead.

As a good rule of thumb, strive to maintain one source of truth - over time, the excerpt above will change and keeping it up to date here is an additional maintainability burden. In the worst case, the excerpt here is not updated and thus the FAQ will have outdated / potentially misleading information - let's prevent that scenario from being possible.

Reply:
Strongly suggest removing this section entirely. It exists entirely to re-explain part of the AWS docs, and thus risks becoming out of date. Additionally, it makes assumptions about details like when the token refresh actually happens that may not be actually true.

Reply:
The reason I included the excerpt is that it was the most helpful explanation of the token bucket system that I've seen, and this system is one that is shared between AWS and K8s. The explanation of the system won't go out of date (But I agree that mentions of specific API names might go out of date and those at a minimum should be removed).

However, I can see an argument for: "We can trust the Cx to figure out how the token bucket algorithm works, especially if we hand them a link to the relevant documentation"

Therefore I'll delete the excerpt.